tests/test_config.py

```diff
@@ -1,32 +1,50 @@
 import os
 
+potential_paths = [
+    'tests/',
+    '../tests/'
+    'tap-square/tests/',
+    '../tap-square/tests/',
+]
+
+def go_to_tests_directory():
+    for path in potential_paths:
+        if os.path.exists(path):
+            os.chdir(path)
+            return os.getcwd()
+    raise NotImplementedError("This check cannot run from {}".format(os.getcwd()))
```

Comment:
Again I really think if done correctly this should be able to run from anywhere but I'm ok with leaving this for now.

Reply:
I agree, I just couldn't figure out how to get that working.